Bigtable/src/V2/BigtableCellBuffer.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Google\Cloud\Bigtable\V2;
+
+use Google\Cloud\Bigtable\V2\Exception\InvalidChunkException;
+
+class BigtableCellBuffer
+{
+    const NEW_CELL = 0;
+    const IN_PROGRESS = 1;
+    const COMPLETE = 2;
+
+    private $rowKey;
+    private $familyName;
+    private $columnName;
+    private $timestampMicros;
+    private $valueBuffer;
+    private $isCommitRowCell;
+    private $state;
+
+    public function __construct()
+    {
+        $this->rowKey = null;
+        $this->familyName = null;
+        $this->columnName = null;
+        $this->resetValueBuffer();
+    }
+
+    /**
+     * Merge a cell chunk into this buffer.
```

Comment:
Nit: Google style mostly recommend the doc to start with a third-person verb, e.g., "Merges".